pkg/scheduler/scheduler.go

```diff
@@ -23,6 +23,36 @@ import (
 	"k8s.io/utils/clock"
 )
 
+// We are writting our own time.AfterFunc to be able to mock the clock. The
+// cancel function can be called concurrently.
```

Comment:
```suggestion
// We are writing our own time.AfterFunc to be able to mock the clock. The
// cancel function can be called concurrently.
```

How come this function moved?

Reply:
Has it also changed?

Reply:
This PR is just the result of:

```bash
git revert 641960b666e94b8f592652f14f7b1fc313c7e97b
```

(I checked it was the same)

The original commit moved the function, so the revert moves it back.